tests/Common/TokensShareTest.php

```diff
@@ -21,8 +21,12 @@ public function testMasterTokenShouldNotBeShareable()
 
     public function testTokenShare()
     {
+        $this->initEvents();
```

Comment:
@tomasfejfar tady bacha, ted to ma povinny parametr `Client $client`, takze tam hod `$this->_client`.

Reply:
Ajo, dyť jsem to viděl v review 🤦

Reply:
upraveno